Doc/library/asyncio-stream.rst

```diff
@@ -50,40 +52,74 @@ Stream functions
    <coroutine>`; if it is a coroutine function, it will be automatically
    converted into a :class:`Task`.
 
-   The rest of the arguments are all the usual arguments to
-   :meth:`~AbstractEventLoop.create_server()` except *protocol_factory*; most
-   common are positional *host* and *port*, with various optional keyword
-   arguments following.
+   When specified, the *loop* argument determines which event loop to use,
+   and the *limit* argument determines the buffer size limit used by the
+   :class:`StreamReader` instance passed to *client_connected_cb*.
 
-   Additional optional keyword arguments are *loop* (to set the event loop
-   instance to use) and *limit* (to set the buffer limit passed to the
-   :class:`StreamReader`).
+   The rest of the arguments are passed directly to
+   :meth:`~AbstractEventLoop.create_server()`.
 
    This function is a :ref:`coroutine <coroutine>`.
 
-.. coroutinefunction:: open_unix_connection(path=None, \*, loop=None, limit=None, **kwds)
+   .. versionadded:: 3.7
+
+      The *ssl_handshake_timeout* and *start_serving* parameters.
+
+.. coroutinefunction:: open_unix_connection(path=None, \*, loop=None, limit=None, ssl=None, sock=None, server_hostname=None, ssl_handshake_timeout=None)
 
    A wrapper for :meth:`~AbstractEventLoop.create_unix_connection()` returning
    a (reader, writer) pair.
 
-   See :func:`open_connection` for information about return value and other
-   details.
+   When specified, the *loop* argument determines which event loop to use,
+   and the *limit* argument determines the buffer size limit used by the
+   returned :class:`StreamReader` instance.
+
+   The rest of the arguments are passed directly to
+   :meth:`~AbstractEventLoop.create_unix_connection()`.
 
    This function is a :ref:`coroutine <coroutine>`.
 
    Availability: UNIX.
 
-.. coroutinefunction:: start_unix_server(client_connected_cb, path=None, \*, loop=None, limit=None, **kwds)
+   .. versionadded:: 3.7
+
+      The *ssl_handshake_timeout* parameter.
+
+   .. versionchanged:: 3.7
+
+      The *path* parameter can now be a :term:`path-like object`
+
+.. coroutinefunction:: start_unix_server(client_connected_cb, path=None, \*, loop=None, limit=None, sock=None, backlog=100, ssl=None, ssl_handshake_timeout=None, start_serving=True)
 
    Start a UNIX Domain Socket server, with a callback for each client connected.
 
-   See :func:`start_server` for information about return value and other
-   details.
+   The *client_connected_cb* parameter is called with two parameters:
```

Comment:
Perhaps add a few words for better reader understanding:

`The callback for a connected client, *client_connected_cb*, contains a *client_reader* and *client_writer*.`